### PR TITLE
Readme update

### LINKED ISSUE #17: README needs an update
**Describe the bug**
SSIA

**To Reproduce**
1. Go to repo
2. Click on README.md
3. See error

**Expected behavior**
a concise description of what this repo is about.

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by bjoernh2000**
## This introduces a breaking change

- [x] No

## This Pull Request implements

Updated readme to reflect project description

## Description

Closes: #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,5 @@
...
 
-1. GitHub defaults and Templates for issues.
-2. Configuration for Coala and Black (code formating) are done using [`prow`](https://github.com/kubernetes/test-infra/tree/master/prow) with `pre-commit` and `mypy`.
-3. Zuul is no longer suppported. Instead [AICOE-CI](https://github.com/AICoE/aicoe-ci) is used for basic configuration.
-4. We have a public service configuration now accessible for everyone.
-5. If you are writing a Python module, [Kebechet](https://github.com/marketplace/khebhut) could manage the releases of your packages for you.
-6. If credentials are provided, Python module releases could be published to [`PyPI`](https://pypi.org/) by [Kebechet](https://github.com/marketplace/khebhut).
-
-Dependencies should be managed using `pipenv` (`Pipfile`, and the `Pipfile.lock` could be created by `thamos advise`), `pip3` and a `requirements.txt` files could be used.
+Dependencies are managed using `pipenv`.
```
Comment (pacospace) [inline README.md]: They are managed by Kebechet bot actually, soon with thoth resolver :)

**Comment by goern** [comment]
:100: 
/lgtm

File: README.md [hunk 2]
```diff
@@ -1,12 +1,5 @@
...
 
-1. GitHub defaults and Templates for issues.
-2. Configuration for Coala and Black (code formating) are done using [`prow`](https://github.com/kubernetes/test-infra/tree/master/prow) with `pre-commit` and `mypy`.
-3. Zuul is no longer suppported. Instead [AICOE-CI](https://github.com/AICoE/aicoe-ci) is used for basic configuration.
-4. We have a public service configuration now accessible for everyone.
-5. If you are writing a Python module, [Kebechet](https://github.com/marketplace/khebhut) could manage the releases of your packages for you.
-6. If credentials are provided, Python module releases could be published to [`PyPI`](https://pypi.org/) by [Kebechet](https://github.com/marketplace/khebhut).
-
-Dependencies should be managed using `pipenv` (`Pipfile`, and the `Pipfile.lock` could be created by `thamos advise`), `pip3` and a `requirements.txt` files could be used.
+Dependencies are managed by the kebechet bot, soon with thoth resolver.
```
Comment (pacospace) [inline README.md]: no need to add `soon with thoth resolver` :)

**Review by pacospace — COMMENTED** [review]
/lgtm

**Comment by bjoernh2000** [comment]
> /assign @goern @bjoernh2000 for approval

Can I approve my own PR?

**Comment by pacospace** [comment]
> @pacospace: GitHub didn't allow me to assign the following users: for, approval.
> 
> Note that only [thoth-station members](https://github.com/orgs/thoth-station/people), repo collaborators and people who have commented on this issue/PR can be assigned. Additionally, issues/PRs can only have 10 assignees at the same time.
> For more information please see [the contributor guide](https://git.k8s.io/community/contributors/guide/first-contribution.md#issue-assignment-in-github)

@goern isn't @bjoernh2000 approver in OWNERS file?

**Comment by bjoernh2000** [comment]
> > @pacospace: GitHub didn't allow me to assign the following users: for, approval.
> > Note that only [thoth-station members](https://github.com/orgs/thoth-station/people), repo collaborators and people who have commented on this issue/PR can be assigned. Additionally, issues/PRs can only have 10 assignees at the same time.
> > For more information please see [the contributor guide](https://git.k8s.io/community/contributors/guide/first-contribution.md#issue-assignment-in-github)
> 
> @goern isn't @bjoernh2000 approver in OWNERS file?

I think sesheta meant that user "for" and user "approval" aren't real users haha

**Comment by goern** [comment]
> @goern isn't @bjoernh2000 approver in OWNERS file?

idk, have a look :)

**Pull request merged**